atom/browser/api/atom_api_web_contents.h

```diff
@@ -6,13 +6,17 @@
 #define ATOM_BROWSER_API_ATOM_API_WEB_CONTENTS_H_
 
 #include <string>
+#include <vector>
+#include <set>
```

Comment:
Hmm this is strange, which line was it complaining? The `atom_api_web_contents.cc` should still need to include `set` though.